api/src/main/java/org/apache/commons/rdf/api/RDFTermFactory.java

```diff
@@ -60,34 +54,28 @@ default BlankNode createBlankNode() throws UnsupportedOperationException {
     }
 
     /**
-     * Create a blank node based on the given identifier.
-     *
-     * All BlankNodes created using this method with the same parameter for a
-     * single instance of RDFTermFactory MUST be equivalent. Ie,
-     * {@link BlankNode#equals(Object)} MUST return true. A BlankNode object
-     * created through the {@link RDFTermFactory#createBlankNode()} method must
-     * be universally unique, and SHOULD contain a {@link UUID} as part of its
-     * {@link BlankNode#internalIdentifier()}.
-     *
-     * A BlankNode object created through the
-     * {@link RDFTermFactory#createBlankNode(String)} method must be universally
-     * unique, but also produce the same {@link BlankNode#internalIdentifier()} as any
-     * previous or future calls to that method on that factory with the same
-     * parameters. In addition, it SHOULD contain a {@link UUID} as part of its
-     * {@link BlankNode#internalIdentifier()}, created using
-     * {@link UUID#nameUUIDFromBytes(byte[])} using a constant salt for each
-     * instance of {@link RDFTermFactory}, with the given identifier joined to
-     * that salt in a consistent manner.
-     *
-     * BlankNodes created using this method with the same parameter, for
-     * different instances of RDFTermFactory, SHOULD NOT be equivalent.
-     *
-     * @param identifier A non-empty, non-null, String that is unique to this blank
-     *                   node in the context of this {@link RDFTermFactory}.
-     * @return A BlankNode for the given identifier
-     * @throws UnsupportedOperationException If the operation is not supported.
+     * Create a blank node based on the given name.
+     * <p>
+     * For a particular instance of <code>RDFTermFactory</code>, all
+     * {@link BlankNode}s created using this method with the same
+     * <code>name</code> parameter MUST be equivalent according to
+     * {@link BlankNode#equals(Object)},
+     * The returned BlankNode MUST NOT be equal to any other
```

Comment:
This sentence could be cleaned up. In particular, it could be emphasised that it only applies when the sentence above does not apply. Ie, when the name parameter is different, the BlankNode object must be not equals.